noxfile.py

```diff
@@ -113,6 +113,14 @@ def update_proj(session: nox.Session) -> None:
     )
 
 
+@nox.session(reuse_venv=True)
+def generate_schema(session: nox.Session) -> None:
+    session.install("pyyaml", "ruamel.yaml")
```

Comment:
I think that ruamel.yaml isn't used?
```suggestion
    session.install("pyyaml")
```